apps/src/codegen.js

```diff
@@ -488,6 +531,8 @@ function createSelection (selection, cumulativeLength, start, end) {
   // calling with the backwards parameter set to true - this prevents horizontal
   // scrolling to the right while stepping through in the debugger
   selection.setSelectionRange(range, true);
+  highlightAceLines(aceEditor, highlightClass || "ace_step", range.start.row,
+      range.end.row);
```

Comment:
We seem to do `highlightClass || "ace_step"` a few times. Would it make more sense to have highlightAceLines take care of defaulting to ace_step?